### PR TITLE
Assingment3, Assignment4 & HandsOn

### PR DESCRIPTION
**Pull request by aelxfun200**
I upload all because the Pull request for Assigment3 was not closed.

### CODE REVIEW

**Comment by ocorcho** [comment]
Assignment 3 is ok. Now @paoespinozarias will check assignment 2b, @dachafra will check assignment 4 and @rgcmme will check group assignment 1

**Comment by rgcmme** [comment]
The self-assessment file is not filled. It is just a matter of putting an x between the brackets for those items satisfied.

**Review by dachafra — CHANGES_REQUESTED** [review]
6.3: Create a new individual of Researcher named "Jane Smith" is not correct, it is asking to create a new instance of Researcher, please change it
T7.2 and T7.3 are also wrong, prefixes are not used correctly, and queries do not make sense, take a look to contributions from your colleagues

File: Assignment2/aelxfun200-fromRDFtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+					"@id":"http://www.oeg-upm.net/ontologies/class/Class01/Sensor029",
+					"@type":"vocab_2:Sensor",
+					"temperatura":"29",
+					"fech_hora":"2010-06-12T12:00:12"
+			}
+
+		}
+	}
+
+}
```
Comment (paoespinozarias) [inline Assignment2/aelxfun200-fromRDFtoJSON-LD.jsonld:38]: The relationships "includes" between a Sensor and a Computer are missing. Also, the relationship "hasMeasurement" is missing. Please, take a look at the assignments that have been already accepted and make the needed changes.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by aelxfun200** [comment]
Cabe mencionar que en los ficheros de HandsOn hemos contribuido todos aunque haya subido yo los archivos directamente a mi repositorio.

**Pull request closed without merging**